### PR TITLE
Adding support for ARM64

### PR DESCRIPTION
**Pull request by ashruti-msft**
Adding support for ARM64

### CODE REVIEW

File: blobfuse2-release.yaml [hunk 1]
```diff
@@ -142,7 +142,108 @@ stages:
...
+            workingDirectory: $(root_dir)
+            displayName: 'Accumulate pkg files'
+          
+          # using fpm tool for packaging of our binary & performing post-install operations
+          # for additional information about fpm refer https://fpm.readthedocs.io/en/v1.13.1/
+          - script: |
+              fpm -s dir -t deb -n blobfuse2 -C pkgDir/ -v `./pkgDir/usr/bin/blobfuse2 --version | cut -d " " -f 3` -d $(depends) \
+              --maintainer "Blobfuse v-Team <blobfusevteam@microsoft.com>" --url "https://github.com/Azure/azure-storage-fuse" \
+              --description "An user-space filesystem for interacting with Azure Storage" 
+              mv ./blobfuse2*.deb ./blobfuse2-`./pkgDir/usr/bin/blobfuse2 --version | cut -d " " -f 3`-$(tags).aarch64.deb
```
Comment (vibhansa-msft) [inline blobfuse2-release.yaml:227]: for deb file rename this to ".arm64.deb" and not ".aarch64.deb"

File: blobfuse2-release.yaml [hunk 2]
```diff
@@ -142,7 +142,108 @@ stages:
             inputs:
               artifactName: 'blobfuse2-temp'
             displayName: 'Publish Artifacts' 
-  # BuildArtifacts end here
+
+      - job: Set_1_ARM
```
Comment (souravgupta-msft) [inline blobfuse2-release.yaml:146]:
```suggestion
      - job: Set_2_ARM
```

File: blobfuse2-release.yaml (hunk 1 above)
Comment (souravgupta-msft) [inline blobfuse2-release.yaml:227]:
```suggestion
              mv ./blobfuse2*.deb ./blobfuse2-`./pkgDir/usr/bin/blobfuse2 --version | cut -d " " -f 3`-$(tags).arm64.deb
```

File: blobfuse2-release.yaml [hunk 3]
```diff
@@ -392,26 +497,121 @@ stages:
...
               artifactName: 'blobfuse2-signed'
               downloadPath: $(root_dir)
-              itemPattern: blobfuse2-signed/blobfuse2*$(tags)*.deb
+              itemPattern: blobfuse2-signed/blobfuse2*$(tags).xdeb
 
           - script: |
-              for f in ./blobfuse2*$(tags)*.deb; do mv -v "$f" "${f/-$(tags)./-$(vmImage).}"; done;
-              cp ./blobfuse2*$(vmImage)*.deb $(Build.ArtifactStagingDirectory)
+              for f in ./blobfuse2*$(tags)x86_64.deb; do mv -v "$f" "${f/-$(tags)./-$(vmImage).}"; done;
+              cp ./blobfuse2*$(vmImage)*x86_64.deb $(Build.ArtifactStagingDirectory)
```
Comment (souravgupta-msft) [inline blobfuse2-release.yaml:504]: Remove this and the changes below relate to `x86_64` as the rename is already done in the above line.
```suggestion
              cp ./blobfuse2*$(vmImage)*.deb $(Build.ArtifactStagingDirectory)
```

File: blobfuse2-release.yaml [hunk 4]
```diff
@@ -392,26 +497,121 @@ stages:
             inputs:
               artifactName: 'blobfuse2-signed'
               downloadPath: $(root_dir)
-              itemPattern: blobfuse2-signed/blobfuse2*$(tags)*.deb
+              itemPattern: blobfuse2-signed/blobfuse2*$(tags).xdeb
```
Comment (souravgupta-msft) [inline blobfuse2-release.yaml:500]:
```suggestion
              itemPattern: blobfuse2-signed/blobfuse2*$(tags).deb
```

File: cmd/health-monitor_test.go [hunk 5]
```diff
@@ -68,6 +71,19 @@ type hmonTestSuite struct {
 	assert *assert.Assertions
 }
 
+func generateRandomPID() string {
+	rand.Seed(time.Now().UnixNano())
+	var randpid int
+	for i := 0; i <= 5; i++ {
+		randpid = rand.Intn(9000) + 1000
```
Comment (souravgupta-msft) [inline cmd/health-monitor_test.go:78]: Make the random pid length as 5

```suggestion
		randpid = rand.Intn(90000) + 10000
```

**Review by vibhansa-msft — APPROVED** [review]
LGTM

File: blobfuse2-nightly.yaml [hunk 6]
```diff
@@ -198,6 +198,143 @@ stages:
...
 
+        - job: Set_1_ARM
+          timeoutInMinutes: 300
+
+          strategy:
+            matrix:
+              Ubuntu-22-ARM64-BlockBlob:
+                imageName: 'blobfuse-ubn22-arm64'
+                containerName: 'test-cnt-ubn-22-arm64'
+                adlsSas: $(AZTEST_ADLS_CONT_SAS_UBN_22)
```
Comment (souravgupta-msft) [inline blobfuse2-nightly.yaml:209]: SAS needs to point to the arm64 container SAS.

**Review by souravgupta-msft — APPROVED** [review]

File: blobfuse2-nightly.yaml (hunk 6 above)
Comment (souravgupta-msft) [reply blobfuse2-nightly.yaml:209]:
```suggestion
                adlsSas: $(AZTEST_ADLS_CONT_SAS_UBN_22_ARM)
```

**Pull request merged**